libs/framework/src/dm_dependency_manager_impl.c

```diff
@@ -173,7 +174,8 @@ static void celix_dm_getInfosCallback(void *handle, const celix_bundle_t *bnd) {
 
 celix_array_list_t * celix_dependencyManager_createInfos(celix_dependency_manager_t *manager) {
 	celix_array_list_t *infos = celix_arrayList_create();
-	celix_bundleContext_useBundles(manager->ctx, infos, celix_dm_getInfosCallback);
+	celix_framework_t* fw = celix_bundleContext_getFramework(manager->ctx);
```

Comment:
Why this change?

Reply:
In de pubsub deadlock test the dependency manager is used from the framework bundle context. This works, but the 'dm' shell command was not printing info about the dm stuff of the framework. 
This changes fixes that (using a useBundles call which includes the framework "bundle".

Reply:
Ok, the dm is explicitly part of the fw? Since this now added a direct include to the framework which was not needed previously.

Reply:
yes the dm is part of the framework. This removed the need for additional linking (per bundle) and more importantly the needed special linking instructions with the "-Wl,--no-as-needed" for linux and for osx something else.